Developer Samples/WindowsContainers/docker-compose.yml

```diff
@@ -0,0 +1,23 @@
+version: '2.4'
+
+services: 
+    rex:
+        depends_on:
+            - cat
+        image: inrule/inrule-runtime:v5.1.1
```

Comment:
Will hardcording this to v5.1.1 be a problem when future versions are released? Or is this a sample where the expectation is that someone who downloads this sample will change on their own?

Reply:
Good question -- I know that the official Guidance is not to use "latest" tag in docker-compose or other orchestration definitions, so I went with an explicit version tag. I think it may be possible to replace the version with an environment or similar variable. 

Regarding the docs update - good catch on that. I'll push an update that removes/fixes those references

Reply:
It makes sense regarding the docker-compose tags and thank you clarifying. I defer to your judgement. We could update this sample as we push out future releases.

Once the documentation is updated, this PR should be good to go.

Reply:
Updates pushed. There are a number of different areas that needed updating, but I think I got them all